terraform/core/main.tf

```diff
@@ -68,14 +68,35 @@ data "archive_file" "cloud-function-src" {
   type        = "zip"
   source_dir  = "${path.root}/../../server"
   output_path = "${path.root}/../../generated/gcf-src.zip"
-  excludes = [ "__pycache__", "static" ]
+  excludes = [
+    "__pycache__",
+    "app_engine/.app_engine_ignore",
+    "app_engine/announcements.py",
+    "app_engine/app.yaml",
+    "app_engine/app_engine.py",
+    "app_engine/config.py",
+    "app_engine/credentialstore.py",
+    "app_engine/dataset_producer.py",
+    "app_engine/dataset_zipper.py",
+    "app_engine/env_variables.yaml",
+    "app_engine/oidc.py",
+    "app_engine/requirements.txt",
+    "app_engine/roles.py",
+    "app_engine/team_info.py",
+    "app_engine/test_routes.py",
+    "app_engine/tracking.py",
+    "app_engine/wrappers.py",
+    "src",
+    "static",
+    "templates"
```

Comment:
The reorganization described above could make this a tad cleaner by only having to exclude app_engine itself, not all of the individual files?  This would also make maintenance easier as you wouldn't have to remember to edit this if you add a file to app_engine.